pyfdb/pyfdb.py

```diff
@@ -260,10 +260,17 @@ class FDB:
     """This is the main container class for accessing FDB"""
     __fdb = None
 
-    def __init__(self):
+    def __init__(self, config_path=None):
         fdb = ffi.new('fdb_handle_t**')
-        lib.fdb_new_handle(fdb)
-
+        if config_path is None:
+            lib.fdb_new_handle(fdb)
+        else:
+            if type(config_path) is str:
```

Comment:
My instinct is that this is not the right way to do this. I think we probably want to support the Config object (or at the least a LocalConfiguration) natively in the C API, and accept that as an argument.

We also need to handle the userConfiguration correctly.

@geier1993 - can you recall what (if any) support exists for the LocalConfig object via C APIs?

Reply:
There is no C API for eckit AFAIK.